dll/win32/shell32/dialogs/drvdefext.cpp

```diff
@@ -238,6 +238,7 @@ CDrvDefExt::PaintStaticControls(HWND hwndDlg, LPDRAWITEMSTRUCT pDrawItem)
 
             MoveToEx(pDrawItem->hDC, x, y, NULL);
             LineTo(pDrawItem->hDC, x, y + 10);
+			SetPixel(pDrawItem->hDC, x, y + 10, RGB(0, 0, 0));
```

Comment:
Why was this SetPixel added?

Reply:
Added pictures Without_SetPixel.png and With_SetPixel.png to the JIRA issue to help explain.

Reply:
So you do basically say that this line is part of the vertical stretching.
Why aren't the changes with the changed height in the resources enough for that? from
{code}
CONTROL "", 14015, "Static", SS_OWNERDRAW, 70, 116, 100, 30
{code}
to
{code}
CONTROL "", 14015, "Static", SS_OWNERDRAW, 70, 116, 100, 40
{code}
I guess that is what [~learn_more]s question was about.

Reply:
It's just to add the bottom outline, a resource alone wont add it.